src/markdown/home.mdx

```diff
@@ -48,6 +48,6 @@ An indie game, or independent video game, is a video game that is often created
 
 ## How do I contribute?
 
-#### Want to submit a game or check out submitted games? [Submit your game!](/games)
+#### Want to submit a game or check out [submitted games](/games)? [Submit your game!](/developers)
```

Comment:
Thanks for the catch! We just rearranged some things... 
Can you change this to 
```
#### [Submit your game](/developers) or check out [submitted games](/games)!

Reply:
No problem! I almost did it that way, but didn't want to change your copy.